libr/bin/bfile.c

```diff
@@ -82,6 +94,13 @@ static int string_scan_range(RList *list, RBinFile *bf, int min,
 		free (buf);
 		return -1;
 	}
+	st64 vdelta = 0;
+	if (bf->o) {
```

Comment:
Is there any valid reason for a binfile to not have a binobject? It seems a r-return-Val at the beginning of the function would be better